google-cloud-firestore/src/main/java/com/google/cloud/firestore/DocumentReference.java

```diff
@@ -354,6 +354,19 @@ public T apply(List<T> results) {
     return extractFirst(firestore.getAll(this));
   }
 
+  /**
+   * Reads the document referenced by this DocumentReference. If the document doesn't exist, the
+   * get(FieldMask fieldMask) will return an empty DocumentSnapshot.
+   *
```

Comment:
As googlers say, the specification requires present tense 

If the document doesn't exist, the get(FieldMask fieldMask) will return an empty DocumentSnapshot.
->
If the document doesn't exist an empty DocumentSnapshot is returned.